spec/controllers/external_authors_controller_spec.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe ExternalAuthorsController do
+  include LoginMacros
+  include RedirectExpectationHelper
+  let(:user) { FactoryGirl.create(:user) }
+  let(:invitation) { FactoryGirl.create(:invitation, external_author: external_author) }
+  let(:external_author) { FactoryGirl.create(:external_author) }
+
+  before(:each) do
```

Comment:
Like I said, I'm really not sure what the best course is -- you might want to ask for a second opinion